po/de.po

```diff
@@ -18869,17 +18841,16 @@ msgstr "Konnte Objekt '%s' nicht parsen."
 msgid "hash mismatch %s"
 msgstr "Hash stimmt nicht mit %s überein."
 
-#, fuzzy, c-format
+#, c-format
 msgid "duplicate entry when writing bitmap index: %s"
-msgstr "duplizierter Eintrag im Bitmap-Index: '%s'"
+msgstr "doppelter Eintrag beim Schreiben des Bitmap-Index: %s"
 
 #, c-format
 msgid "attempted to store non-selected commit: '%s'"
-msgstr ""
+msgstr "versuchte, nicht gewählten Commit '%s' zu speichern"
```

Comment:
mit dem Doppelpunkt in dem originalen String wäre ich mir jetzt nicht so sicher, dass '%s' für den Commit steht.. vielleicht ist es ja noch eine weitere Fehlermeldung, die dort ausgegeben wird oder so.. 🤔